CHANGELOG.md

```diff
@@ -2,87 +2,63 @@
 
 All notable changes to this project will be documented in this file.
 
-<!--attr-start-->
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-
-<!--attr-end-->
-
-<!--
-Using the following categories, list your changes in this order:
+## [Unreleased]
 
 ### Added
--   for new features.
-
-### Changed
--   for changes in existing functionality.
-
-### Deprecated
--   for soon-to-be removed features.
-
-### Removed
--   for removed features.
-
-### Fixed
--   for bug fixes.
-
-### Security
--   for vulnerability fixes.
- -->
-
-<!--changelog-start-->
-
-## [Unreleased](https://github.com/Archmonger/ServeStatic/compare/2.0.1...HEAD)
 
--   Nothing (yet)!
+- New CI workflow, including utilizing PyPI trusted publishing for releases and creation of GitHub releases
+- Formatting and linting is now provided by `ruff`
 
-## [2.0.1](https://github.com/Archmonger/ServeStatic/compare/2.0.0...2.0.1) - 2024-09-13
```

Comment:
I'm okay with moving the links down, but I originally moved them inline with the version numbers to prevent myself from forgetting to change them.

Reply:
I'll have to do some checking.  The action to parse this seemed quite strict about the formatting, with no options to lessen it.

Side note that with Github releases, there will be buttons there to diff at least, though it is an extra step.

Reply:
Resolved this

Reply:
Hmmm, feels like a misnomer though. The new "full changelog" link technically doesn't even lead to a changelog, just a commit history.

I think we can just shove the links at the bottom of the page, and I can a small linting script for us that will throw an error if someone forgot to update a link.

Reply:
Done